lib/govuk_app_config/govuk_content_security_policy.rb

```diff
@@ -29,12 +29,8 @@ def self.build_policy(policy)
     policy.default_src :self
 
     # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src
+    # Note: we purposely don't include data here because it produces a security risk.
```

Comment:
Suggestion for clarity:

```suggestion
    # Note: we purposely don't include `data:` here because it produces a security risk.
```